src/docker.test.ts

```diff
@@ -87,14 +87,15 @@ describe("Docker images", (): void => {
   const assertSaveDockerImages = (
```

Comment:
Commit message: "save the cache" --> "attempt to save the cache" and "unnecessary cache saves" --> "unnecessary cache save attempts" since GitHub Actions doesn't allow you to overwrite a cache key without deleting the previous cache first.